snowflake/JsonUploadFromBlob.txt

```diff
@@ -0,0 +1,78 @@
+-- SELECT DATABASE --
+use database BLOB_DB;
+
+use schema PUBLIC;
+
+use role accountadmin;
+
+-- Create stage --
+
+create or replace stage azure_stage_sas
+  URL='azure://<storage-account>.blob.core.windows.net/<container>'
+  CREDENTIALS=(AZURE_SAS_TOKEN='<sas-token>');
+
+list @azure_stage_sas;
+
+-- Import raw data to table --
+
+create or replace table RAW_SOURCE (
+  SRC variant, V_SHA varchar(100));
+
+create or replace task copy_data

```

Comment:
This task name could be a little bit more descriptive (e.g. `copy_data_from_azure`)